bundles/persistence/org.openhab.persistence.sitewhere/.classpath

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<classpath>
+	<classpathentry kind="src" path="src/main/java"/>
+	<classpathentry kind="src" path="src/main/resources"/>
+	<classpathentry exported="true" kind="lib" path="lib/spring-beans-3.2.10.RELEASE.jar"/>
+	<classpathentry exported="true" kind="lib" path="lib/spring-context-3.2.10.RELEASE.jar"/>
+	<classpathentry exported="true" kind="lib" path="lib/spring-context-support-3.2.10.RELEASE.jar"/>
+	<classpathentry exported="true" kind="lib" path="lib/spring-core-3.2.10.RELEASE.jar"/>
+	<classpathentry exported="true" kind="lib" path="lib/spring-web-3.2.10.RELEASE.jar"/>
+	<classpathentry exported="true" kind="lib" path="lib/httpclient-4.3.1.jar"/>
+	<classpathentry exported="true" kind="lib" path="lib/httpcore-4.3.jar"/>
+	<classpathentry exported="true" kind="lib" path="lib/jackson-annotations-2.1.5.jar"/>
+	<classpathentry exported="true" kind="lib" path="lib/jackson-core-2.1.5.jar"/>
+	<classpathentry exported="true" kind="lib" path="lib/jackson-databind-2.1.5.jar"/>
```

Comment:
is httpclient 4.3.1 needed? A smaller version should be available through the target platform. Same applies to jackson